Gemfile

```diff
@@ -3,6 +3,10 @@ source "https://rubygems.org"
 # Specify your gem's dependencies in capistrano.gemspec
 gemspec
 
+gem "mocha"
+gem "rspec"
+gem "rspec-core", "~> 3.4.4"
```

Comment:
I moved our remaining dev dependencies out of the gemspec and into the Gemfile. This is the pattern now recommended by bundler. I.e. gemspec is for runtime deps, Gemfile is for dev deps.